src/components/PeopleFilters/PeopleFilters.tsx

```diff
@@ -0,0 +1,132 @@
+import classNames from 'classnames';
+import { useSearchParams } from 'react-router-dom';
+import { SearchLink } from '../SearchLink';
+import { Sex } from '../../types/Sex';
+
+const CENTURIES_ARRAY = [16, 17, 18, 19, 20];
+
+export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const query = searchParams.get('query') || '';
+  const centuries = searchParams.getAll('centuries') || [];
+
+  function handleQueryChange(event: React.ChangeEvent<HTMLInputElement>) {
```

Comment:
It is important to follow the same way of creating functions, fix it everywhere
```suggestion
  const handleQueryChange = (event: React.ChangeEvent<HTMLInputElement>) => {
```